### ISSUE TITLE
some orig content getting erroneously capitalized

### ISSUE THREAD

**Issue by paregorios** [DCLP/dclpxsltbox#272]
Fixes for issue #243 were intended to force capitalizations of all "clear-but-incomprehensible letters"; however, some other cases involving different semantics are getting capitalized as well. 

For example, see the second apparatus line in HTML for TM 112359, lines 3-4:
![screen shot 2017-05-22 at 2 09 09 pm](https://cloud.githubusercontent.com/assets/263285/26325147/618d8220-3efb-11e7-95ff-d661fc774d27.png)

where ΣΤΗΚΕΙ is inappropriately capitalized.

**Comment by paregorios** [comment]
The code in question, in teiorig.xsl, includes a test that looks like it should isolate ```<orig>``` not inside choice; however, it is either not working as expected, or is not getting invoked, in apparatus cases such as this example.

**Comment by wsalesky** [comment]
Should I edit the branch for this issue: https://github.com/DCLP/dclpxsltbox/issues/243

Or should I wait for you to merge it into the trunk?

**Comment by paregorios** [comment]
The updates are already in the DCLP-production branch (^/branches/dclp/example-p5-stylesheets), unless I'm horribly confused.

**Comment by wsalesky** [comment]
Hum, must be a problem on my side. I use svn up . to pull in changes, and am at rv: 2633 but do not see the change to teiorig.xsl. Let me try again.

**Comment by paregorios** [comment]
@wsalesky something is wrong with my rebase-and-merge process in this case

**Comment by wsalesky** [comment]
**XSLT changed:** epidoc-xslt/teiorig.xsl
Added better testing for` tei:orig `with` tei:choice` parent elements. 

**Branch:** svn.code.sf.net/p/epidoc/code/branches/dclp-issue243/example-p5-xslt

**Comment by paregorios** [comment]
@wsalesky this last change must have got stomped by my failed merge process ... see https://github.com/DCLP/dclpxsltbox/issues/118#issuecomment-303877406

I think you'll have to recreate the fix. Apologies.

**Comment by wsalesky** [comment]
**See branch for the fix:** https://github.com/DCLP/epidoc-xslt/tree/issue272

**Comment by wsalesky** [comment]
**XSLT:** ` epidoc-xslt/example-p5-xslt/tpl-apparatus.xsl`
Added special for `t:orig[parent::t:choice]` handling to named template `"recurse_down_back"`

**HTML changed:** 
`output/dclp/113/112359.html`

**HTML diff:**
```
- <span>|[ΣΤΗΚΕΙ]ϊ
+ <span>|[στηκει]ϊ
```

While this works, I feel this is a patch that may unnecessarily complicates the code. Refactoring the tpl-apparatus is a non trivial undertaking.
